src/Test/Utilities/Portable/Compilation/TestOperationVisitor.cs

```diff
@@ -1122,6 +1122,28 @@ public override void VisitDeclarationPattern(IDeclarationPatternOperation operat
             Assert.Empty(operation.Children);
         }
 
+        public override void VisitRecursivePattern(IRecursivePatternOperation operation)
+        {
```

Comment:
>{ [](start = 8, length = 1)

Please explicitly assert that Type and ConstantValue are not set for nodes representing patterns